src/components/modals/AddressModal.tsx

```diff
@@ -330,9 +330,9 @@ export class AddressModalComponent extends React.Component<Props, State> {
             error={fieldError}
           />
           {!userFioAddressesLoading ? (
-            <FlatList
+            <FlashList
               data={filteredFioAddresses}
-              initialNumToRender={24}
+              estimatedItemSize={72}
```

Comment:
Please use `theme.rem(4.25)` for this, as all our sizes depend on that scale factor. Most of our rows use a standard height, like 3 or 4 rem, which you can determine from the row component itself.